pretty_midi/instrument.py

```diff
@@ -51,11 +53,15 @@ def __init__(self, program, is_drum=False, name=''):
         """
         self.program = program
         self.is_drum = is_drum
+        self.is_vocals = False
         self.name = name
         self.notes = []
         self.pitch_bends = []
         self.control_changes = []
 
+    def set_as_vocals(self):
```

Comment:
The convention in pretty_midi is to not include getter/setter functions and to instead allow public manipulation of instance attributes (e.g. the user should just do Instrument.is_vocals = True instead of calling Instrument.set_as_vocals()). So, I would delete this method.